### PR TITLE
More UTC fixes for asset events

### PR DESCRIPTION
**Pull request by Bobfrat**
@birdage @DanielJMaher please review/merge

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@c0mster You made some changes so that this would work in FireFox.. .does any of Bob's work step on this?

**Comment by birdage** [comment]
@Bobfrat @DanielJMaher we should be consistent with date time reps. in the system i.e 'YYYY-MM-DD HH:mm:ss'?, dont we want to use the same format?

File: ooiui/static/js/views/asset_management/EventsView.js [hunk 1]
```diff
@@ -108,8 +107,7 @@ var EventsView = Backbone.View.extend({
             formatter: _.extend({}, Backgrid.CellFormatter.prototype, {
               fromRaw: function (rawValue, model) {
                 if(rawValue !=null){
-                  var sd = new Date(rawValue);
-                  return sd.toDateString();
+                  return moment.utc(rawValue).format("ddd MMM DD YYYY");
```
Comment (birdage) [inline ooiui/static/js/views/asset_management/EventsView.js]: @Bobfrat here?

File: ooiui/static/js/views/asset_management/EventsView.js [hunk 2]
```diff
@@ -92,8 +92,7 @@ var EventsView = Backbone.View.extend({
...
```
Comment (birdage) [inline ooiui/static/js/views/asset_management/EventsView.js]: @Bobfrat here?

**Comment by Bobfrat** [comment]
@birdage we're now consistent with timezone (UTC) but some places we are just displaying time in a table so the different formats look nicer. The date pickers are all consistent ISO. @DanielJMaher up to you

**Comment by DanielJMaher** [comment]
I don't mind having different time representations for now.

**Comment by birdage** [comment]
@Bobfrat @c0mster this needs to get tested some more with all the other changes in place.

**Comment by DanielJMaher** [comment]
@Bobfrat Can you check it in firefox?

**Comment by c0mster** [comment]
See http://stackoverflow.com/questions/3257460/new-date-is-working-in-chrome-but-not-firefox
Firefox has strict guidelines for using new Date(.... moment is the best alternative

**Comment by Bobfrat** [comment]
Agreed! We should all use moment and UTC. Theres a half dozen different ways we're displaying time right now in this code base.

**Comment by Bobfrat** [comment]
Dates look good in firefox, but I noticed the page ```events/list/``` is rendering all mangled. @c0mster can you take a look at this one?

**Comment by c0mster** [comment]
Sure can

**Pull request merged**